sys/linux/socket_netlink_generic_80211.txt

```diff
@@ -11,12 +11,16 @@ include <linux/ieee80211.h>
 # But it makes sense to setup hwsim devices first, otherwise there is not much to test.
 
 resource nl80211_family_id[int16]
+resource nl80211_ifindex[int32]
+
+nl80211_devnames = "wlan0", "wlan1", "wlan2"
 
 type msghdr_nl80211[CMD, POLICY] msghdr_netlink[netlink_msg_t[nl80211_family_id, genlmsghdr_t[CMD], POLICY]]
 
 type nl80211_whipy int32[-1:4]
 
 syz_genetlink_get_family_id$nl80211(name ptr[in, string["nl80211"]]) nl80211_family_id
+ioctl$sock_SIOCGIFINDEX_80211(fd sock, cmd const[SIOCGIFINDEX], arg ptr[inout, ifreq_dev_t[nl80211_devnames, nl80211_ifindex[opt]]])
```

Comment:
Not directly related to your change as we do it this way everywhere. But @necipfazil recently added field direction to structs, and this looks like a perfect use case for them. We declare these ifindex'es as opt only because they are actually out, while the ifr_ifrn field in ifreq is in. We could specify directions directly now.

Reply:
Adjusted `ifreq_dev_t` and its uses in a separate commit